packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -16,14 +16,18 @@ import { isLWorAF, taggingNamePluralSetting } from '../../lib/instanceSettings';
 import stringify from 'json-stringify-deterministic';
 import { isFriendlyUI } from '../../themes/forumTheme';
 import { FRIENDLY_HOVER_OVER_WIDTH } from '../common/FriendlyHoverOver';
+import { AnnualReviewMarketInfo, highlightMarket } from '../../lib/annualReviewMarkets';
 
 const styles = (theme: ThemeType) => ({
   root: {
-    marginTop: 8,
-    marginBottom: 8,
+    display: 'flex',
```

Comment:
I _think_ that all the styling changes here were @Discordius.  Same q as above - was this intended to apply for all uses of `FooterTagList` on LW, and if so, do we want to forum-gate them?